packages/spans/src/index.ts

```diff
@@ -0,0 +1,199 @@
+import { NativeModules } from 'react-native';
+import { Attributes, Events, SPAN_ERROR_CODES } from '../interfaces/ISpans';
+import {
+  convertMSToNano,
+  createFalsePromise,
+  validateAndLogRequiredProperties,
+} from './Utils';
+
+export const startSpanWithName = (
+  name: string,
+  parentSpanId?: string
+): Promise<boolean | string> => {
+  if (!validateAndLogRequiredProperties({ name })) {
+    return createFalsePromise();
+  }
+  try {
+    return NativeModules.EmbraceManager.startSpanWithName(name, parentSpanId);
+  } catch (e) {
+    console.warn(
+      `[Embrace] The method startSpanWithName was not found, please update the SDK.`,
+      e
+    );
+    return createFalsePromise();
+  }
+};
+
+export const stopSpanWithId = (
+  spanId: string,
+  errorCode: SPAN_ERROR_CODES = 'None'
+): Promise<boolean> => {
+  if (!validateAndLogRequiredProperties({ spanId })) {
+    return createFalsePromise();
+  }
+  try {
+    return NativeModules.EmbraceManager.stopSpanWithId(spanId, errorCode);
+  } catch (e) {
+    console.warn(
+      `[Embrace] The method stopSpanWithId was not found, please update the SDK.`,
+      e
+    );
+    return createFalsePromise();
+  }
+};
+
+export const addSpanEventToSpanId = (
+  spanId: string,
+  name: string,
+  timeStamp?: number,
```

Comment:
Do we usually add the unit (e.g. ms) to parameter names by convention? If it's ok, timeStampMs is preferred since the OTel JavaScript SDK uses nanos and we don't, so making that clear would be nice

This applies to all timestamps BTW.